_information/online_giving.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
I want a big giving button on this page to be the same as the "Donate" link on the nav bar.

Reply:
Let's move it above that title.
Can we center it?

Reply:
uh-oh... That didn't work: 
<img width="494" height="308" alt="image" src="https://github.com/user-attachments/assets/bb67ce48-8118-4564-b5a2-bac929171c8a" />